docker/bin/zookeeperReady.sh

```diff
@@ -27,40 +28,62 @@ if [[ "$OK" == "imok" ]]; then
   set +e
   nslookup $DOMAIN
   if [[ $? -eq 1 ]]; then
-
     set -e
     echo "There is no active ensemble, skipping readiness probe..."
     exit 0
-
   else
     set -e
     # An ensemble exists, check to see if this node is already a member.
     # Check to see if zookeeper service for this node is a participant
     set +e
-    ZKURL=$(zkConnectionString)
-    set -e
-    MYID=`cat $MYID_FILE`
-    ROLE=`java -Dlog4j.configuration=file:"$LOG4J_CONF" -jar /root/zu.jar get-role $ZKURL $MYID`
+    # Extract resource name and this members ordinal value from pod hostname
```

Comment:
Typo: `member ordinal value`

Reply:
This was an existing comment, will change it nevertheless....

Reply:
I think it should be the `members' ordinal value`

Reply:
Updated